packages/e2e/README.md

```diff
@@ -113,6 +113,39 @@ Address:    addr_test1qr0c3frkem9cqn5f73dnvqpena27k2fgqew6wct9eaka03agfwkvzr0zyq
 
 You can configure any of these five wallets in your test and use any amount of tADA you need.
 
+### Local Test Network - development version
+
+In addition to **local-network** there is its development version, which mounts the `packages` directory in the
+http-server container and in the worker container. It helps to apply changes to the two components on a running
+**local-network** whithout the need to shutdown and restart it.
+
+It can be started with:
+
+```bash
+$ yarn workspace @cardano-sdk/e2e local-network:dev
+```
+
+and a good idea is to run following command before starting it.
```

Comment:
I would make this a `prelocal-network:dev` script, then remove this manual step.